src/test/java/com/puppycrawl/tools/checkstyle/checks/NoCodeInFileCheckTest.java

```diff
@@ -0,0 +1,85 @@
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks;
+
+import static com.puppycrawl.tools.checkstyle.checks.NoCodeInFileCheck.MSG_KEY_NO_CODE;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport;
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+import com.puppycrawl.tools.checkstyle.utils.CommonUtil;
+
+public class NoCodeInFileCheckTest extends AbstractModuleTestSupport {
+
+    @Override
+    protected String getPackageLocation() {
+        return "com/puppycrawl/tools/checkstyle/checks/nocodeinfile";
+    }
+
+    @Test
+    public void testGetRequiredTokens() {
+        final NoCodeInFileCheck checkObj = new NoCodeInFileCheck();
+        assertArrayEquals(CommonUtil.EMPTY_INT_ARRAY, checkObj.getRequiredTokens(),
+                "Required tokens array is not empty");
+    }
+
+    @Test
+    public void testGetAcceptableTokens() {
+        final NoCodeInFileCheck checkObj = new NoCodeInFileCheck();
+        assertArrayEquals(CommonUtil.EMPTY_INT_ARRAY, checkObj.getAcceptableTokens(),
+                "Acceptable tokens array is not empty");
+    }
+
+    @Test
+    public void testBlank() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(NoCodeInFileCheck.class);
+        final String[] expected = {
+            "1: " + getCheckMessage(MSG_KEY_NO_CODE),
+        };
+        verify(checkConfig, getPath("InputNoCodeInFile1.java"), expected);
+    }
+
+    @Test
+    public void testSingleLineComment() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(NoCodeInFileCheck.class);
+        final String[] expected = {
+            "1: " + getCheckMessage(MSG_KEY_NO_CODE),
+        };
+        verify(checkConfig, getPath("InputNoCodeInFile2.java"), expected);
+    }
+
+    @Test
+    public void testMultiLineComment() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(NoCodeInFileCheck.class);
+        final String[] expected = {
+            "1: " + getCheckMessage(MSG_KEY_NO_CODE),
+        };
+        verify(checkConfig, getPath("InputNoCodeInFile3.java"), expected);
+    }
+
+    @Test
+    public void testFileContainingCode() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(NoCodeInFileCheck.class);
+        verify(checkConfig, getPath("InputNoCodeInFile4.java"),
+                CommonUtil.EMPTY_STRING_ARRAY);
+    }
```

Comment:
can be added one more test with both single and multiline comments

Reply:
Done.